pkgs/applications/networking/ids/suricata/default.nix

```diff
@@ -0,0 +1,131 @@
+{ stdenv
+, lib
+, fetchurl
+, pkgconfig
+, makeWrapper
+, file
+, geoip
+, hyperscan
+, jansson
+, libcap_ng
+, libevent
+, libnet
+, libnetfilter_log
+, libnetfilter_queue
+, libnfnetlink
+, libpcap
+, libyaml
+, luajit
+, nspr
+, nss
+, pcre
+, python
+, zlib
+, redisSupport ? true, redis, hiredis
+, rustSupport ? true, rustc, cargo
+}:
+
+let
+
+  libmagic = file;
+  hyperscanSupport = stdenv.system == "x86_64-linux" || stdenv.system == "i686-linux";
+
+in
+stdenv.mkDerivation rec {
+  version = "4.1.4";
+  name = "suricata-${version}";
+
+  src = fetchurl {
+    name = "${name}.tar.gz";
```

Comment:
Is that necessary?

Reply:
what exactly do you mean? yes a source tarball needs to be fetched. but a git url could also be used if that should be what you mean?

Reply:
Is it necessary to pass the name attribute to fetchurl?